src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+    int supplyCounter = 0;
+    int buyCounter = 0;
+        Path path = Path.of(fromFileName);
+        Path pathToSave = Path.of(toFileName);
+        try {
+            String readedFile = String.valueOf(Files.readAllLines(path)).
```

Comment:
Using `String.valueOf(Files.readAllLines(path))` is not the best approach to read file content. This converts the list of strings into a single string representation, which might not be suitable for further processing. Instead, consider joining the lines with a specific delimiter or processing each line individually.